extensions/amp-access-laterpay/0.1/test/validator-amp-access-laterpay.html

```diff
@@ -0,0 +1,71 @@
+<!--
```

Comment:
Sorry for not responding earlier. We currently don't have a mechanism to enforce an attribute across all tags given if another tag exists. It would be possible if the attribute was limited to a single or small set of elements.